quic/fizz/server/handshake/FizzServerQuicHandshakeContext.h

```diff
@@ -20,6 +22,40 @@ class FizzServerQuicHandshakeContext
  public:
   std::unique_ptr<ServerHandshake> makeServerHandshake(
       QuicServerConnectionState* conn) override;
+
+  const std::shared_ptr<const fizz::server::FizzServerContext>& getContext()
```

Comment:
Can this returns const ref? I think the only usage of the return value is to be used to copy contrusct another FizzServerContext.

Reply:
I realize you are following the ownership model of existing code. So the current version is fine as well. I'm just curious if you see a reason returning a const ref won't work.

Reply:
It's only use for a test, so i think either is fine.

Reply:
Well with this diff it is now used in FizzServerHandshake::initializeImpl as well. Anyway, like i said this was before this diff.